src/components/avatar/Avatar.js

```diff
@@ -55,21 +61,28 @@ class Avatar extends React.PureComponent<Props, State> {
     };
 
     render() {
-        const { avatarUrl, className, name, id, size = '' }: Props = this.props;
+        const { avatarUrl, className, name, id, size = '', isPending = false }: Props = this.props;
         const { hasImageErrored }: State = this.state;
         const classes = classNames(['avatar', className, { [`avatar--${size}`]: SIZES[size] }]);
 
+        const hasNoAvatar = !avatarUrl && !isPending;
+        const loadingBackgroundColor = lightGray;
+        const initialsBackgroundColor = `${getColor(id)}`;
+        const backgroundColor = hasNoAvatar || hasImageErrored ? initialsBackgroundColor : loadingBackgroundColor;
+
         let avatar;
-        if (avatarUrl && !hasImageErrored) {
+        if (isPending) {
```

Comment:
Any thoughts on using the initials in pending case if they exist? This would prevent 2 visual states for the initials case

Reply:
@alexkrolick, can you address or respond to the outstanding comments?

Reply:
Working on it. Design signed off on not showing the initials for the loading state - which would entail putting the `<img />` tag on top of the initials in any case.

Reply:
Can you explain more "which would entail putting the <img /> tag on top of the initials"
Like can't you just show initials by default and then make a request to get avatar. When that request comes or fails you replace initials with the image.

Whats the use case around "pending" state of avatars?

Reply:
Is there a path when name is not there but pending true? In other words we cant even show initials?